src/App.js

```diff
@@ -1,25 +1,63 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  NavLink,
+} from 'react-router-dom';
+import AddItem from './components/AddItem';
+import List from './components/List';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div
+        className="App"
+        style={{
+          display: 'flex',
+          flexFlow: 'column nowrap',
+          justifyContent: 'flex-start',
+          alignItems: 'center',
+          minHeight: '100vh',
+          position: 'relative',
+        }}
+      >
+        <h1>Shopping app</h1>
```

Comment:
Great use of heading hierarchy with the `h1` here and the `h2`s in the components.